feature-matrix.json

```diff
@@ -75,7 +76,8 @@
     "field-list": "✅",
     "label": "✅",
     "list-nolabel": "✅",
-    "list": "✅"
+    "list": "✅",
+    "counter": ""
```

Comment:
I had never heard of this appearance before today—first in the meeting doc about design revisions for numeric inputs, now here. I don't see it in the [XLSForm Docs](https://xlsform.org/en/#appearance), but from the meeting doc it sounds like the expectation is implemented (at least partly? increment/decrement buttons). Although I guess we'd want to explicitly account for (== implement it) it so we don't regress if we decide to limit that presentation for inputs without a `counter` appearance. I'm fine with leaving it unchecked here if that's the intent, but wanted to note my surprise at its addition.

I also think this emphasizes my caution about treating this as a starting point for building out some sort of programmatic form feature support detection. In this case, it isn't just a gap in what we're tracking. It also implicates cross-referencing at least one other project beyond the XLSForm Docs. I see it was [added relatively recently](https://github.com/getodk/collect/pull/6306), but now I also see other appearances there which aren't in the XLSForm Docs and weren't added as recently.

Reply:
It was in the original`integer` issue and I probably should have at least mentioned it in review: https://github.com/getodk/web-forms/issues/251 I think we need to figure out the design for the no-appearance question type and then go from there.

The authoritative sources of appearances are the XLSForm template and the ODK docs. 

If you do see something missing from somewhere, please file an issue!

Reply:
Ah, I totally glazed over it in #251! I saw the reference to Enketo's presentation and probably stopped there. (Likely because the increment/decrement presentation is a de facto default on the web, as it's the default presentation for `<input type="number">` in many cases.)